### ISSUE TITLE
opencv_core.DMatchVector misses some iterating methods

### ISSUE THREAD

**Issue by Jindil** [bytedeco/javacpp#241]
The [API documentation](http://bytedeco.org/javacpp-presets/opencv/apidocs/org/bytedeco/javacpp/opencv_core.DMatchVector.html) states that the DMatchVector object has the method `begin` and `end` that returns some iterator.  
But in practice those methods are not visible (in Jython at least). Maybe they are not useful, but then they should be removed from the API doc .
Here with a jython script :  

```python
from org.bytedeco.javacpp.opencv_core import DMatchVector

Vector = DMatchVector()
print Vector
print dir(Vector)
```
returns
```
org.bytedeco.javacpp.opencv_core$DMatchVector[address=0x3ab948a0,position=0,limit=1,capacity=1,deallocator=org.bytedeco.javacpp.Pointer$NativeDeallocator[ownerAddress=0x3ab948a0,deallocatorAddress=0x7ffd2f375320]]
['__class__', '__copy__', '__deepcopy__', '__delattr__', '__doc__', '__ensure_finalizer__', '__eq__', '__format__', '__getattribute__', '__hash__', '__init__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__str__', '__subclasshook__', '__unicode__', 'address', 'asBuffer', 'asByteBuffer', 'capacity', 'class', 'close', 'deallocate', 'deallocateReferences', 'equals', 'fill', 'get', 'getClass', 'hashCode', 'isNull', 'limit', 'memchr', 'memcmp', 'memcpy', 'memmove', 'memset', 'notify', 'notifyAll', 'null', 'offsetof', 'position', 'put', 'resize', 'setNull', 'size', 'sizeof', 'toString', 'wait', 'zero']
```

**Comment by saudet** [comment]
I'm pretty sure they work from Java, have you tried?

**Comment by Jindil** [comment]
I don't know much about java, I have tried `import org.bytedeco.javacpp.opencv_core.DMatchVector;`
but I get a null pointer exception. I guess I am missing some line before.
Just to precise, I am using javacpp within the [script editor](https://github.com/scijava/script-editor) of the Fiji distribution.

**Comment by saudet** [comment]
Make sure to use a recent version of the presets at least.

**Comment by Jindil** [comment]
Hmm right, the version that is shipped is 1.1, I will stick to it as it could create some conflicts with the package manager of fiji.
Yet if no one has the issue with the last version i guess we can close this.
Thanks and sorry for bothering
